tasks/Logger.cpp

```diff
@@ -61,46 +63,15 @@ Logger::~Logger()
 
 bool Logger::startHook()
 {
-    if(_file.value().empty())
-    {
-      log(Error) << "Could not create log file. Task property _file is empty." << endlog();
-      return false;
-    }
+    std::string currentFile;
 
-    if (_overwrite_existing_files.get() && _auto_timestamp_files.get())
-    {
-        log(Error) << "The properties overwrite_existing_files and auto_timestamp_files are both set to true, but are mutually exclusive." << endlog();
+    if (!computeCurrentFile(_file.value(), currentFile)){
         return false;
     }
+    auto_ptr<ofstream> io(new ofstream(currentFile.c_str()));
+    _current_file.set(currentFile);
+    updateLoggers(io);
```

Comment:
This is essentially the same logic than `setFile`. Is there a reason to not simply call `setFile` ?